Source/ConsoleGUI/Menu.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace ConsoleGUI
 {
-    public class Menu
+    public static class Menu
```

Comment:
We have changed so that almost none other class and method is non static. Regarding this class I did'nt think it was nesseccary to change. And also VS- recommended to have it static. 

> Bra hanterat ifall pjäsen går förbi mållinjen.
> Kanske lite långt i program.cs, överväg att flytta ut delar till nya klasser.
> Bra jobbat, Kämpa på!

Tack så mycket! Gällande att det är mycket kod i program.cs är för att vi väntar med att flytta ut det sista tills vi har grunden klar